README.md

```diff
@@ -10,6 +10,10 @@
 <br/>
 </div>
 
+## Documentation
+
+For in-depth information and guidance, please refer to the [SPV Wallet Documentation](https://bsvblockchain.gitbook.io/docs).
+
```

Comment:
As @dorzepowski commented in the other repo:

> It actually should be somewhere below Table Of contents,

Reply:
To clarify that the link provides documentation about the entire toolbox rather than a specific module (such as the js-client in this instance), I would change the sentence to something like:

"For comprehensive information and guidance on the SPV Wallet Toolbox (...)"

Reply:
Added

Reply:
I think PR is empty now